codalab/apps/web/templates/web/competitions/_submit_results_page.html

```diff
@@ -140,6 +144,24 @@
 {% endif %}
 
 <script>
+
+    var docker_image_key = window.location.href + '_docker_image'
+
+    $(document).ready(function () {
+        $(window).unload(saveSettings);
+        loadSettings();
+    });
+
+    function loadSettings() {
```

Comment:
annoying nitpick: I've been trying to do all new variables/functions in JS lowercase_separated_by_underscores.

(another annoying nitpick:  no semicolons  :))